src/games/rcll/utils.clp

```diff
@@ -484,3 +484,34 @@
 	(bind ?update-str (str-cat ?update-str ")"))
   (return ?update-str)
 )
+
+(deffunction confval-to-pb-type (?t)
+  (bind ?type nil)
+  (if (eq ?t UINT) then (bind ?type INTEGER))
+  (if (eq ?t FLOAT) then (bind ?type FLOAT))
+  (if (eq ?t BOOL) then (bind ?type BOOLEAN))
+  (if (eq ?t STRING) then (bind ?type BOOLEAN))
+  (if (eq ?type nil) then
+    (printout warn "try to add public config with unknown type " ?t " -> use STRING instead" crlf)
+    (bind ?type STRING)
+  )
+  (return ?type)
+)
+
+(deffunction tag-ids-from-machine-name (?machine-name)
```

Comment:
shouldn't we move this to a config file too and do a lookup from there?

Reply:
we could but it just introduces yet another load of yaml files that pretty much never get touched anyways, right?
We changed the tags once in the last 8 years, hence I think there is no need to make it configurable.